### ISSUE TITLE
[BUG] - 'browser' object has no attribute 'verifyFp'

### ISSUE THREAD

**Issue by demalem** [davidteather/TikTok-Api#237]
When running the example code from the README:

from TikTokApi import TikTokApi
api = TikTokApi()

results = 10

trending = api.trending(count=results)

for tiktok in trending:
    # Prints the text of the tiktok
    print(tiktok['desc'])

print(len(trending))

I recieve the below error:

AttributeError: 'browser' object has no attribute 'verifyFp'

I installed chromedriver, but may not have added it to the right path. Can you give guidance in this regards?

**Comment by davidteather** [comment]
What happens when you type the following command into your terminal
```
chromedriver
```

**Comment by demalem** [comment]
`'chromedriver' is not recognized as an internal or external command,
operable program or batch file.`

However, I downloaded chromedriver and included in the path where the script is located. Both in C:\Users\emela\.spyder-py3

**Comment by demalem** [comment]
moved chromerdriver to the parent folder where Anaconda is downloaded and got this result now:

`Starting ChromeDriver 85.0.4183.83 (94abc2237ae0c9a4cb5f035431c8adfb94324633-refs/branch-heads/4183@{#1658}) on port 9515
Only local connections are allowed.
Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping ChromeDriver safe.
[1598556547.109][SEVERE]: bind() returned an error: Only one usage of each socket address (protocol/network address/port) is normally permitted. (0x2740)
IPv6 port not available. Exiting...`

**Comment by davidteather** [comment]
You should be including the script in the system path not the script path not a specific directory

**Comment by demalem** [comment]
Same error, I put chromedriver in C:\Users\emela and pointed all my scripts their as well.

**Comment by davidteather** [comment]
[Here's](https://helpdeskgeek.com/windows-10/add-windows-path-environment-variable/) an article on how to add to system path

**Comment by demalem** [comment]
Great thanks, I added it to the PATH variable. Here is a screenshot: 
![image](https://user-images.githubusercontent.com/70233103/91490397-623f6080-e880-11ea-8877-fdf5ed965f98.png)

Restarted kernel, and same error. I also tried running code from the Chromedriver homepage: https://sites.google.com/a/chromium.org/chromedriver/getting-started and ran into issues as well. I'm assuming I'm not correctly including chromedriver in the PATH or pointing to it accurately.

**Comment by wangxy524** [comment]
i met this problem too. Centos 7, already installed Chromedriver and works well. but when i run the api, I recieve the below error:
AttributeError: 'browser' object has no attribute 'verifyFp'
Error in atexit._run_exitfuncs:
raise RuntimeError('Event loop is closed')
sys:1: RuntimeWarning: coroutine 'Launcher.killChrome' was never awaited

**Comment by edenhikri** [comment]
I have tried all of these resolutions and nothing seems to work. I am not in a country in which TikTok is banned. I have also tried running it with debug mode enabled api = TikTokApi(debug=True) however the issue still occurs. After printing 20 users stats, it gives me the error. It seems as though it gets stuck. 

The error:
`Traceback (most recent call last):
  File "snowball.py", line 24, in <module>
    tiktok = api.getUser(accounts)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/TikTokApi/tiktok.py", line 723, in getUser
    return self.getData(b, proxy=proxy)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/TikTokApi/tiktok.py", line 71, in getData
    query = {'verifyFp': b.verifyFp, 'did': b.did, '_signature': b.signature}
AttributeError: 'browser' object has no attribute 'verifyFp'`

my code script (snowball.py)
`from TikTokApi import TikTokApi

api = TikTokApi()

# file_data is the list of 150 tiktok users accounts) 

for accounts in file_data:
	tiktok = api.getUser(accounts)
	name = (tiktok['userInfo']['user']['uniqueId'])
	followers = (tiktok['userInfo']['stats']['followerCount'])


	if followers > 1000000:
		user_videos = api.byUsername(name, count=10)

		for video in user_videos:
			stats = (video['stats'])
			print(stats, name)`

**Comment by davidteather** [comment]
@edenhikri does #246 solve your issue?

**Comment by edenhikri** [comment]
No, unfortunately it does not.

**Comment by edenhikri** [comment]
This may help you as I have seen you request this on other similar issues. When I run:
```
from pyppeteer.launcher import Launcher
print(' '.join(Launcher().cmd))
```

I get this:

/Users/edenhikri/Library/Application Support/pyppeteer/local-chromium/588429/chrome-mac/Chromium.app/Contents/MacOS/Chromium --disable-background-networking --disable-background-timer-throttling --disable-breakpad --disable-browser-side-navigation --disable-client-side-phishing-detection --disable-default-apps --disable-dev-shm-usage --disable-extensions --disable-features=site-per-process --disable-hang-monitor --disable-popup-blocking --disable-prompt-on-repost --disable-sync --disable-translate --metrics-recording-only --no-first-run --safebrowsing-disable-auto-update --enable-automation --password-store=basic --use-mock-keychain --headless --hide-scrollbars --mute-audio about:blank --remote-debugging-port=51908 --user-data-dir=/Users/edenhikri/Library/Application Support/pyppeteer/.dev_profile/tmpi4lv6pb2

**Comment by fastandslow007** [comment]
seems like you have too many request and then get blocked by tiktok server. 
have you try using proxy?

**Comment by edenhikri** [comment]
I am not sure if this is the issue... sometimes it works and does all the accounts, sometimes it gets stuck on the error... @fastandslow007

**Comment by fastandslow007** [comment]
First, try to debug:

```
..
TikTokApi(debug=True)
..

```
Then enable dumpio on launch: <your-env>/lib/python3.6/site-packages/pyppeteer/launcher.py:
```
..
 81         self.dumpio = options.get('dumpio', True)
..

```

Run your script and notice the error. In my case, I got this error:
```
...
Class initialized
<xx>/local-chromium/588429/chrome-linux/chrome: error while loading shared libraries: libXss.so.1: cannot open shared object file: No such file or directory
The following error occurred, but it was ignored.
'browser' object has no attribute 'timezone_name'
<xx>/.local/share/pyppeteer/local-chromium/588429/chrome-linux/chrome: error while loading shared libraries: libXss.so.1: cannot open shared object file: No such file or directory
...
[ERROR]   ERROR-2 'browser' object has no attribute 'verifyFp'
...
```

Try to fix missing libXss.so.1 by installing this (I use Ubuntu):

`$ apt-get install libxss1
`

Then the problem is fixed.

Thanks.

**Comment by edenhikri** [comment]
I have tried this...

I first enabled debug mode
```
api = TikTokApi(debug=True)
```

then I went into /Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pyppeteer/launcher.py
and changed line 81:
```
self.dumpio = options.get('dumpio', True)
```

then I ran snowball.py (my script)

It ran everything properly until it got stuck and brought up the same error:
```
**...**
DevTools listening on ws://127.0.0.1:53029/devtools/browser/28433110-68c9-4304-875e-5494698a533d
[0916/130613.905933:INFO:CONSOLE(1)] "%c", source:  (1)
Traceback (most recent call last):
  File "snowball.py", line 22, in <module>
    tiktok = api.getUser(accounts, proxy=None)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/TikTokApi/tiktok.py", line 723, in getUser
    return self.getData(b, proxy=proxy)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/TikTokApi/tiktok.py", line 71, in getData
    query = {'verifyFp': b.verifyFp, 'did': b.did, '_signature': b.signature}
AttributeError: 'browser' object has no attribute 'verifyFp'
```

**Comment by fastandslow007** [comment]
Is it okay if you share the full log from the start?

**Comment by edenhikri** [comment]
Yes however it is quite lengthy as I am printing out stats for the last 10 videos of each user

**Comment by edenhikri** [comment]
This is what happens when I run snowball.py (python3 snowball.py) with debug mode set to True and dumpio enabled on launch:

```
Class initialized
[0916/134801.418284:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:53033/devtools/browser/bc92f083-e2cf-4b56-848e-9d42eccb9ddd
[0916/134802.035779:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:53286/devtools/browser/b98c0105-3ca1-4618-81c2-c332947a53ea
[0916/134802.487838:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134803.274329:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:53290/devtools/browser/2f293b3d-d766-4d0b-a995-79339d7bc775
[0916/134803.707993:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134804.595638:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:53544/devtools/browser/e52914c9-4260-4ed5-a093-6c13504c6560
[0916/134805.047445:INFO:CONSOLE(1)] "%c", source:  (1)
imjoeyreed {'diggCount': 723300, 'shareCount': 1334, 'commentCount': 18600, 'playCount': 5000000}
imjoeyreed {'diggCount': 3100000, 'shareCount': 10400, 'commentCount': 66700, 'playCount': 18500000}
imjoeyreed {'diggCount': 6100000, 'shareCount': 25700, 'commentCount': 63100, 'playCount': 35500000}
imjoeyreed {'diggCount': 3500000, 'shareCount': 10900, 'commentCount': 33600, 'playCount': 23600000}
imjoeyreed {'diggCount': 737100, 'shareCount': 1803, 'commentCount': 8723, 'playCount': 7300000}
imjoeyreed {'diggCount': 678200, 'shareCount': 1054, 'commentCount': 8930, 'playCount': 6500000}
imjoeyreed {'diggCount': 827200, 'shareCount': 2577, 'commentCount': 13000, 'playCount': 8700000}
imjoeyreed {'diggCount': 6900000, 'shareCount': 68300, 'commentCount': 28500, 'playCount': 43200000}
imjoeyreed {'diggCount': 1600000, 'shareCount': 4595, 'commentCount': 6614, 'playCount': 12100000}
imjoeyreed {'diggCount': 352700, 'shareCount': 1025, 'commentCount': 575, 'playCount': 4200000}
[0916/134805.857978:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:53548/devtools/browser/5251781e-98fa-4b1a-91a5-c1b1274fa889
[0916/134806.292298:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134807.290627:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:53802/devtools/browser/0bc4581c-5cac-4280-bf69-1fb1c0fb7b24
[0916/134807.717116:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134808.512582:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:54056/devtools/browser/41ddcfe0-b6c2-471d-ab12-b3d4777d9c28
[0916/134808.962879:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134809.744046:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:54060/devtools/browser/ba1cc5ef-f1c3-4a16-b3bb-66291b474b33
[0916/134810.168042:INFO:CONSOLE(1)] "%c", source:  (1)
jorobe {'diggCount': 192600, 'shareCount': 2304, 'commentCount': 19, 'playCount': 550800}
jorobe {'diggCount': 63100, 'shareCount': 1462, 'commentCount': 11, 'playCount': 260800}
jorobe {'diggCount': 140600, 'shareCount': 1653, 'commentCount': 12, 'playCount': 453700}
jorobe {'diggCount': 203600, 'shareCount': 2986, 'commentCount': 17, 'playCount': 692600}
jorobe {'diggCount': 1000000, 'shareCount': 9930, 'commentCount': 12, 'playCount': 3300000}
jorobe {'diggCount': 125900, 'shareCount': 1034, 'commentCount': 22, 'playCount': 515700}
jorobe {'diggCount': 337500, 'shareCount': 2897, 'commentCount': 12, 'playCount': 1300000}
jorobe {'diggCount': 262000, 'shareCount': 1997, 'commentCount': 16, 'playCount': 921500}
jorobe {'diggCount': 265600, 'shareCount': 1641, 'commentCount': 22, 'playCount': 980300}
jorobe {'diggCount': 617200, 'shareCount': 19300, 'commentCount': 21, 'playCount': 2200000}
[0916/134810.971645:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:54313/devtools/browser/4d0386fd-de84-423a-879a-6f3ccf505ee7
[0916/134811.407280:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134812.202574:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:54317/devtools/browser/8a3561ee-22d8-48b8-9493-d7a79ef92eaa
[0916/134812.636919:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134813.346903:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:54569/devtools/browser/c8e978ab-1a53-475a-b8e0-c578c07224e4
[0916/134813.777412:INFO:CONSOLE(1)] "%c", source:  (1)
richcaroline {'diggCount': 39700, 'shareCount': 104, 'commentCount': 696, 'playCount': 123800}
richcaroline {'diggCount': 1600000, 'shareCount': 8939, 'commentCount': 6410, 'playCount': 5200000}
richcaroline {'diggCount': 1600000, 'shareCount': 15900, 'commentCount': 4910, 'playCount': 6100000}
richcaroline {'diggCount': 217000, 'shareCount': 488, 'commentCount': 1005, 'playCount': 882300}
richcaroline {'diggCount': 513200, 'shareCount': 1283, 'commentCount': 1758, 'playCount': 1700000}
richcaroline {'diggCount': 744600, 'shareCount': 2687, 'commentCount': 3162, 'playCount': 2500000}
richcaroline {'diggCount': 393800, 'shareCount': 4742, 'commentCount': 2758, 'playCount': 1400000}
richcaroline {'diggCount': 607700, 'shareCount': 1747, 'commentCount': 2479, 'playCount': 1900000}
richcaroline {'diggCount': 427400, 'shareCount': 1629, 'commentCount': 1901, 'playCount': 1600000}
richcaroline {'diggCount': 1700000, 'shareCount': 7034, 'commentCount': 4739, 'playCount': 6000000}
[0916/134814.660344:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:54823/devtools/browser/72578646-ac64-46d1-b5eb-6cd5ca5e5c4f
[0916/134815.106695:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134815.987528:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:54827/devtools/browser/01b4a282-a28f-4196-abb5-79a7ba0609b0
[0916/134816.410309:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134817.377603:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:55081/devtools/browser/4e188ee1-9d50-402b-b741-0127535aea9a
[0916/134817.841818:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134818.722272:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:55337/devtools/browser/8fe26923-3248-4c30-8fdf-31d020c06f10
[0916/134819.201214:INFO:CONSOLE(1)] "%c", source:  (1)
thehousenobodyaskedfor {'diggCount': 1900000, 'shareCount': 3455, 'commentCount': 6782, 'playCount': 6600000}
thehousenobodyaskedfor {'diggCount': 1200000, 'shareCount': 1832, 'commentCount': 3371, 'playCount': 4500000}
thehousenobodyaskedfor {'diggCount': 881800, 'shareCount': 5669, 'commentCount': 7070, 'playCount': 7400000}
thehousenobodyaskedfor {'diggCount': 1200000, 'shareCount': 1786, 'commentCount': 4360, 'playCount': 4300000}
thehousenobodyaskedfor {'diggCount': 1500000, 'shareCount': 2517, 'commentCount': 10700, 'playCount': 5300000}
thehousenobodyaskedfor {'diggCount': 585800, 'shareCount': 1465, 'commentCount': 3453, 'playCount': 2000000}
thehousenobodyaskedfor {'diggCount': 668300, 'shareCount': 1348, 'commentCount': 10500, 'playCount': 2800000}
thehousenobodyaskedfor {'diggCount': 2400000, 'shareCount': 3668, 'commentCount': 51200, 'playCount': 8300000}
thehousenobodyaskedfor {'diggCount': 3600000, 'shareCount': 6535, 'commentCount': 73700, 'playCount': 13500000}
thehousenobodyaskedfor {'diggCount': 2400000, 'shareCount': 5560, 'commentCount': 101600, 'playCount': 9300000}
[0916/134820.148576:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:55341/devtools/browser/fa1711d4-a9f2-462e-8c80-8376b0bb58d5
[0916/134820.650348:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134821.655398:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:55597/devtools/browser/94326aa5-82c2-43e7-8163-3833e4c0f28d
[0916/134822.108003:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134823.060542:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:55851/devtools/browser/be9f94b8-d368-4bff-9722-45835dba3f7c
[0916/134823.526817:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134824.290284:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:55855/devtools/browser/2f309bab-aa49-40dd-b774-f535886c8576
[0916/134824.719780:INFO:CONSOLE(1)] "%c", source:  (1)
onlyjayus {'diggCount': 105500, 'shareCount': 31600, 'commentCount': 4961, 'playCount': 287500}
onlyjayus {'diggCount': 202600, 'shareCount': 1061, 'commentCount': 1047, 'playCount': 641100}
onlyjayus {'diggCount': 353200, 'shareCount': 3163, 'commentCount': 5722, 'playCount': 996400}
onlyjayus {'diggCount': 436900, 'shareCount': 1822, 'commentCount': 4121, 'playCount': 1300000}
onlyjayus {'diggCount': 77300, 'shareCount': 408, 'commentCount': 946, 'playCount': 339500}
onlyjayus {'diggCount': 367000, 'shareCount': 842, 'commentCount': 5185, 'playCount': 1400000}
onlyjayus {'diggCount': 1600000, 'shareCount': 23700, 'commentCount': 9967, 'playCount': 6900000}
onlyjayus {'diggCount': 718200, 'shareCount': 1607, 'commentCount': 3345, 'playCount': 2300000}
onlyjayus {'diggCount': 582600, 'shareCount': 1279, 'commentCount': 4505, 'playCount': 2400000}
onlyjayus {'diggCount': 1400000, 'shareCount': 10600, 'commentCount': 20400, 'playCount': 5000000}
[0916/134825.536781:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:56109/devtools/browser/a8266b2f-863b-4ba3-b9b6-e45dcbd710c9
[0916/134825.969535:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134826.747753:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:56361/devtools/browser/ab513de5-5b1e-49fb-9bc5-587d7b81ef2b
[0916/134827.225849:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134828.088582:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:56365/devtools/browser/6fc94956-8701-489a-b30b-76d8e2995f8e
[0916/134828.542214:INFO:CONSOLE(1)] "%c", source:  (1)
thereallightskinmonte {'diggCount': 1032, 'shareCount': 8, 'commentCount': 95, 'playCount': 7058}
thereallightskinmonte {'diggCount': 1375, 'shareCount': 5, 'commentCount': 86, 'playCount': 9620}
thereallightskinmonte {'diggCount': 8567, 'shareCount': 32, 'commentCount': 165, 'playCount': 50800}
thereallightskinmonte {'diggCount': 51500, 'shareCount': 919, 'commentCount': 535, 'playCount': 285500}
thereallightskinmonte {'diggCount': 4732, 'shareCount': 31, 'commentCount': 138, 'playCount': 34600}
thereallightskinmonte {'diggCount': 36100, 'shareCount': 106, 'commentCount': 300, 'playCount': 321900}
thereallightskinmonte {'diggCount': 9575, 'shareCount': 93, 'commentCount': 210, 'playCount': 66800}
thereallightskinmonte {'diggCount': 24700, 'shareCount': 82, 'commentCount': 248, 'playCount': 140600}
thereallightskinmonte {'diggCount': 8371, 'shareCount': 91, 'commentCount': 177, 'playCount': 90600}
thereallightskinmonte {'diggCount': 4903, 'shareCount': 46, 'commentCount': 136, 'playCount': 47300}
[0916/134829.731614:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:56619/devtools/browser/3fa82682-9090-4952-8193-aea586242fea
[0916/134830.182898:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134831.052410:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:56871/devtools/browser/354287d7-9f3c-447c-895a-7445ae605546
[0916/134831.539305:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134832.496009:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:56875/devtools/browser/02c5a332-9059-4cf6-82f5-5833990dc57f
[0916/134832.945775:INFO:CONSOLE(1)] "%c", source:  (1)
nicholasthedude {'diggCount': 18400, 'shareCount': 71, 'commentCount': 1407, 'playCount': 91900}
nicholasthedude {'diggCount': 47900, 'shareCount': 158, 'commentCount': 2088, 'playCount': 175000}
nicholasthedude {'diggCount': 144800, 'shareCount': 1634, 'commentCount': 3328, 'playCount': 588800}
nicholasthedude {'diggCount': 167900, 'shareCount': 229, 'commentCount': 2669, 'playCount': 650400}
nicholasthedude {'diggCount': 333100, 'shareCount': 1481, 'commentCount': 5026, 'playCount': 2000000}
nicholasthedude {'diggCount': 65000, 'shareCount': 203, 'commentCount': 1062, 'playCount': 318900}
nicholasthedude {'diggCount': 47100, 'shareCount': 126, 'commentCount': 1568, 'playCount': 248700}
nicholasthedude {'diggCount': 43600, 'shareCount': 62, 'commentCount': 1164, 'playCount': 278700}
nicholasthedude {'diggCount': 187200, 'shareCount': 871, 'commentCount': 2679, 'playCount': 774900}
nicholasthedude {'diggCount': 54400, 'shareCount': 146, 'commentCount': 1886, 'playCount': 262300}
[0916/134833.781581:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:57128/devtools/browser/37d64ab8-3da1-484c-93d7-6171d4092508
[0916/134834.235866:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134835.149461:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:57132/devtools/browser/e13c26d2-30ef-4e21-8759-38a1a5cca2bb
[0916/134835.596854:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134836.476755:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:57385/devtools/browser/6bdb45c5-eab8-4f0a-a615-93945631da87
[0916/134836.920111:INFO:CONSOLE(1)] "%c", source:  (1)
theacefamily {'diggCount': 175500, 'shareCount': 622, 'commentCount': 0, 'playCount': 801500}
theacefamily {'diggCount': 613700, 'shareCount': 5507, 'commentCount': 0, 'playCount': 3500000}
theacefamily {'diggCount': 192100, 'shareCount': 205, 'commentCount': 0, 'playCount': 1100000}
theacefamily {'diggCount': 909200, 'shareCount': 1893, 'commentCount': 0, 'playCount': 6600000}
theacefamily {'diggCount': 1300000, 'shareCount': 22800, 'commentCount': 0, 'playCount': 12100000}
theacefamily {'diggCount': 533500, 'shareCount': 731, 'commentCount': 0, 'playCount': 3700000}
theacefamily {'diggCount': 755600, 'shareCount': 8873, 'commentCount': 0, 'playCount': 6700000}
theacefamily {'diggCount': 806300, 'shareCount': 5053, 'commentCount': 0, 'playCount': 5200000}
theacefamily {'diggCount': 1200000, 'shareCount': 5361, 'commentCount': 0, 'playCount': 7800000}
theacefamily {'diggCount': 692900, 'shareCount': 7605, 'commentCount': 0, 'playCount': 4400000}
[0916/134837.716891:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:57638/devtools/browser/20b1bd9a-5c2b-4eb7-99d4-3378a5477cf6
[0916/134838.164704:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134838.959778:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:57642/devtools/browser/3708256b-b25b-45ef-bc73-9dec412ffcf0
[0916/134839.419631:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134840.146941:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:57896/devtools/browser/83835b22-87d0-4b0f-8de4-bebe0af16696
[0916/134840.603496:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134841.847327:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:57937/devtools/browser/1a89d068-8b85-48a0-99ea-d4e71a4fde22
[0916/134842.315867:INFO:CONSOLE(1)] "%c", source:  (1)
olsennchris {'diggCount': 385500, 'shareCount': 7096, 'commentCount': 6141, 'playCount': 1600000}
olsennchris {'diggCount': 47200, 'shareCount': 88, 'commentCount': 388, 'playCount': 249100}
olsennchris {'diggCount': 11900, 'shareCount': 29, 'commentCount': 220, 'playCount': 123400}
olsennchris {'diggCount': 108700, 'shareCount': 248, 'commentCount': 395, 'playCount': 466000}
olsennchris {'diggCount': 536200, 'shareCount': 10400, 'commentCount': 7892, 'playCount': 2000000}
olsennchris {'diggCount': 137000, 'shareCount': 816, 'commentCount': 865, 'playCount': 733400}
olsennchris {'diggCount': 222500, 'shareCount': 371, 'commentCount': 453, 'playCount': 868500}
olsennchris {'diggCount': 399700, 'shareCount': 2878, 'commentCount': 3330, 'playCount': 1500000}
olsennchris {'diggCount': 358800, 'shareCount': 1638, 'commentCount': 1891, 'playCount': 1600000}
olsennchris {'diggCount': 268100, 'shareCount': 805, 'commentCount': 843, 'playCount': 1100000}
[0916/134843.372165:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:58152/devtools/browser/da0464ff-fe67-486f-8ca6-c147db3c8914
[0916/134843.822770:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134844.594093:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:58405/devtools/browser/e2d7653d-7058-456b-a33c-9b12036c07fa
[0916/134845.064981:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134846.033016:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:58409/devtools/browser/27158fa7-68db-4a1e-9a94-2754cfbbfd67
[0916/134846.519267:INFO:CONSOLE(1)] "%c", source:  (1)
saraecheagaray {'diggCount': 118100, 'shareCount': 103, 'commentCount': 2637, 'playCount': 354900}
saraecheagaray {'diggCount': 134100, 'shareCount': 321, 'commentCount': 1288, 'playCount': 488000}
saraecheagaray {'diggCount': 187000, 'shareCount': 310, 'commentCount': 1181, 'playCount': 746800}
saraecheagaray {'diggCount': 1700000, 'shareCount': 20900, 'commentCount': 17800, 'playCount': 6500000}
saraecheagaray {'diggCount': 526900, 'shareCount': 2560, 'commentCount': 4757, 'playCount': 2500000}
saraecheagaray {'diggCount': 197800, 'shareCount': 3217, 'commentCount': 717, 'playCount': 976400}
saraecheagaray {'diggCount': 145300, 'shareCount': 135, 'commentCount': 391, 'playCount': 993100}
saraecheagaray {'diggCount': 1400000, 'shareCount': 4623, 'commentCount': 6784, 'playCount': 5900000}
saraecheagaray {'diggCount': 354900, 'shareCount': 1329, 'commentCount': 2604, 'playCount': 1800000}
saraecheagaray {'diggCount': 5100000, 'shareCount': 44100, 'commentCount': 49200, 'playCount': 22400000}
[0916/134847.439980:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:58663/devtools/browser/1bf300cd-12af-4556-960e-3dd6a453671f
[0916/134847.917942:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134848.778168:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:58917/devtools/browser/a991630e-12c6-4e2c-ac81-101203ad5830
[0916/134849.266258:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134850.080694:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:58924/devtools/browser/7cf5f9aa-d15a-4e7b-9a67-f4b6c099d10f
[0916/134850.560684:INFO:CONSOLE(1)] "%c", source:  (1)
heyeliza {'diggCount': 16900, 'shareCount': 67, 'commentCount': 218, 'playCount': 67400}
heyeliza {'diggCount': 37500, 'shareCount': 111, 'commentCount': 434, 'playCount': 135500}
heyeliza {'diggCount': 26000, 'shareCount': 111, 'commentCount': 262, 'playCount': 99400}
heyeliza {'diggCount': 23200, 'shareCount': 99, 'commentCount': 365, 'playCount': 115100}
heyeliza {'diggCount': 42400, 'shareCount': 362, 'commentCount': 711, 'playCount': 163500}
heyeliza {'diggCount': 44800, 'shareCount': 103, 'commentCount': 388, 'playCount': 169100}
heyeliza {'diggCount': 221200, 'shareCount': 1304, 'commentCount': 815, 'playCount': 825300}
heyeliza {'diggCount': 43400, 'shareCount': 85, 'commentCount': 255, 'playCount': 192100}
heyeliza {'diggCount': 43100, 'shareCount': 70, 'commentCount': 346, 'playCount': 194200}
heyeliza {'diggCount': 68900, 'shareCount': 240, 'commentCount': 990, 'playCount': 273000}
[0916/134851.549833:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:59176/devtools/browser/0ecdc0be-b4b7-4c54-9d93-c19609618b8f
[0916/134851.980843:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134852.776761:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:59429/devtools/browser/2ee5194d-9600-4bda-94bf-03fea2a6e636
[0916/134853.299042:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134854.193414:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:59434/devtools/browser/a1dbeebf-178a-4bd4-8a13-b804620193b9
[0916/134854.616863:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134855.520615:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:59687/devtools/browser/230d8ed8-2eaa-4abf-9071-ffc485a8ae37
[0916/134855.953575:INFO:CONSOLE(1)] "%c", source:  (1)
noahschnacky {'diggCount': 31300, 'shareCount': 1239, 'commentCount': 2329, 'playCount': 155100}
noahschnacky {'diggCount': 51500, 'shareCount': 49, 'commentCount': 169, 'playCount': 303500}
noahschnacky {'diggCount': 147400, 'shareCount': 753, 'commentCount': 1133, 'playCount': 967300}
noahschnacky {'diggCount': 40700, 'shareCount': 72, 'commentCount': 360, 'playCount': 234500}
noahschnacky {'diggCount': 75000, 'shareCount': 1664, 'commentCount': 938, 'playCount': 614100}
noahschnacky {'diggCount': 93900, 'shareCount': 176, 'commentCount': 279, 'playCount': 372800}
noahschnacky {'diggCount': 30300, 'shareCount': 932, 'commentCount': 1159, 'playCount': 177100}
noahschnacky {'diggCount': 229400, 'shareCount': 998, 'commentCount': 1447, 'playCount': 994400}
noahschnacky {'diggCount': 135200, 'shareCount': 372, 'commentCount': 594, 'playCount': 526000}
noahschnacky {'diggCount': 51100, 'shareCount': 152, 'commentCount': 474, 'playCount': 276600}
[0916/134856.853814:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:59940/devtools/browser/139b3049-5c31-4a8f-ab46-18a544515e4d
[0916/134857.298590:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134858.080305:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:59944/devtools/browser/1c69baf3-67a3-43ed-ac7d-c56c4b633708
[0916/134858.504316:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134859.330368:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:60197/devtools/browser/87325509-4bf1-4f23-adb5-4c200acb3b33
[0916/134859.757233:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134906.070393:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:60451/devtools/browser/a5c84ed9-d37b-485c-bf02-4bf3eaa0d526
[0916/134906.493416:INFO:CONSOLE(1)] "%c", source:  (1)
baidaugh {'diggCount': 5243, 'shareCount': 40, 'commentCount': 306, 'playCount': 22400}
baidaugh {'diggCount': 83800, 'shareCount': 412, 'commentCount': 1323, 'playCount': 328300}
baidaugh {'diggCount': 58900, 'shareCount': 87, 'commentCount': 407, 'playCount': 208000}
baidaugh {'diggCount': 21700, 'shareCount': 318, 'commentCount': 578, 'playCount': 107100}
baidaugh {'diggCount': 129900, 'shareCount': 1567, 'commentCount': 1415, 'playCount': 675100}
baidaugh {'diggCount': 42900, 'shareCount': 85, 'commentCount': 640, 'playCount': 157900}
baidaugh {'diggCount': 62800, 'shareCount': 291, 'commentCount': 779, 'playCount': 285300}
baidaugh {'diggCount': 28900, 'shareCount': 374, 'commentCount': 450, 'playCount': 129200}
baidaugh {'diggCount': 301800, 'shareCount': 1032, 'commentCount': 2979, 'playCount': 1400000}
baidaugh {'diggCount': 9684, 'shareCount': 22, 'commentCount': 132, 'playCount': 76400}
[0916/134912.868453:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:60705/devtools/browser/000b9d69-5337-420d-8e2e-579eac5a5a18
[0916/134913.319543:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134914.269314:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:60958/devtools/browser/52c6218d-ead1-487a-b2ee-010a135fcc87
[0916/134914.692786:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134915.534111:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:61212/devtools/browser/8cff5333-622c-4340-8837-6c56902bc84b
[0916/134915.964310:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134916.933413:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:61216/devtools/browser/0a2dd0e2-6b32-4e95-9851-3c9fb221f48e
[0916/134917.369406:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134918.288689:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:61469/devtools/browser/8b9f5928-1503-4b75-8e84-13f7ce5ef2e6
[0916/134918.718790:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134919.489476:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:61723/devtools/browser/1dc1a9a2-4d58-4a41-8133-4f9b4ceeac50
[0916/134919.939091:INFO:CONSOLE(1)] "%c", source:  (1)
6ix9ine {'diggCount': 938700, 'shareCount': 2933, 'commentCount': 20200, 'playCount': 7800000}
6ix9ine {'diggCount': 656100, 'shareCount': 1493, 'commentCount': 10400, 'playCount': 4700000}
6ix9ine {'diggCount': 929000, 'shareCount': 6030, 'commentCount': 22600, 'playCount': 7500000}
6ix9ine {'diggCount': 799200, 'shareCount': 12300, 'commentCount': 26200, 'playCount': 7800000}
6ix9ine {'diggCount': 211900, 'shareCount': 459, 'commentCount': 4081, 'playCount': 3900000}
6ix9ine {'diggCount': 1000000, 'shareCount': 8662, 'commentCount': 20200, 'playCount': 7700000}
6ix9ine {'diggCount': 500000, 'shareCount': 3478, 'commentCount': 14600, 'playCount': 5500000}
6ix9ine {'diggCount': 668900, 'shareCount': 3315, 'commentCount': 23400, 'playCount': 6600000}
6ix9ine {'diggCount': 753400, 'shareCount': 5636, 'commentCount': 30000, 'playCount': 8400000}
6ix9ine {'diggCount': 1300000, 'shareCount': 14000, 'commentCount': 31300, 'playCount': 11300000}
[0916/134920.752770:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:61728/devtools/browser/8e6275ad-708f-48b8-b069-d50ec8b91708
[0916/134921.190374:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134922.025179:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:61980/devtools/browser/2cbdb23a-99b6-4207-9b7b-10d1759235f9
[0916/134922.537106:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134923.297450:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:61984/devtools/browser/24855dd3-dd8c-4c2d-869d-2e2763fbf8ae
[0916/134923.756924:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134924.607124:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:62237/devtools/browser/0615d379-bcaf-4238-aa6e-91573a009320
[0916/134925.092827:INFO:CONSOLE(1)] "%c", source:  (1)
brookemonk_ {'diggCount': 394300, 'shareCount': 3968, 'commentCount': 7648, 'playCount': 1200000}
brookemonk_ {'diggCount': 518300, 'shareCount': 3555, 'commentCount': 10900, 'playCount': 1700000}
brookemonk_ {'diggCount': 458100, 'shareCount': 545, 'commentCount': 2877, 'playCount': 1700000}
brookemonk_ {'diggCount': 453300, 'shareCount': 475, 'commentCount': 3761, 'playCount': 1600000}
brookemonk_ {'diggCount': 987300, 'shareCount': 596, 'commentCount': 2981, 'playCount': 4100000}
brookemonk_ {'diggCount': 563300, 'shareCount': 2791, 'commentCount': 7093, 'playCount': 2200000}
brookemonk_ {'diggCount': 207400, 'shareCount': 306, 'commentCount': 2055, 'playCount': 936000}
brookemonk_ {'diggCount': 243500, 'shareCount': 935, 'commentCount': 4210, 'playCount': 1000000}
brookemonk_ {'diggCount': 1300000, 'shareCount': 2196, 'commentCount': 14300, 'playCount': 5900000}
brookemonk_ {'diggCount': 164600, 'shareCount': 193, 'commentCount': 1147, 'playCount': 691000}
[0916/134926.167051:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:62490/devtools/browser/6968058c-4ddd-474d-bb32-55b13abdb8d2
[0916/134926.608141:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134927.782406:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:62598/devtools/browser/932e300f-e4c8-44ae-8fd9-e6671025c9de
[0916/134928.212378:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134928.956938:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:62747/devtools/browser/d442cd78-a70c-4dcb-9af7-1aa14a0bb0cc
[0916/134929.390571:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134930.246067:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:63001/devtools/browser/0963766d-85d6-475b-8350-0ef9580ef913
[0916/134930.689350:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134931.684323:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:63255/devtools/browser/cc9fb0d1-62ac-400a-a9d7-b52621c0c03d
[0916/134932.132326:INFO:CONSOLE(1)] "%c", source:  (1)
notcursednat {'diggCount': 317500, 'shareCount': 4931, 'commentCount': 3846, 'playCount': 2000000}
notcursednat {'diggCount': 201300, 'shareCount': 1418, 'commentCount': 1889, 'playCount': 1300000}
notcursednat {'diggCount': 154500, 'shareCount': 981, 'commentCount': 1169, 'playCount': 1200000}
notcursednat {'diggCount': 200700, 'shareCount': 2878, 'commentCount': 2370, 'playCount': 1400000}
notcursednat {'diggCount': 300900, 'shareCount': 3341, 'commentCount': 1611, 'playCount': 1800000}
notcursednat {'diggCount': 161900, 'shareCount': 999, 'commentCount': 832, 'playCount': 1200000}
notcursednat {'diggCount': 147500, 'shareCount': 1509, 'commentCount': 1644, 'playCount': 1300000}
notcursednat {'diggCount': 111800, 'shareCount': 1345, 'commentCount': 779, 'playCount': 844900}
notcursednat {'diggCount': 81100, 'shareCount': 419, 'commentCount': 1043, 'playCount': 770400}
notcursednat {'diggCount': 217600, 'shareCount': 472, 'commentCount': 1182, 'playCount': 1700000}
[0916/134933.118763:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:63259/devtools/browser/16db3ad7-0afd-4b73-92b8-b8fd4fcf633d
[0916/134933.555456:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134937.690594:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:63512/devtools/browser/df6d84b8-bd7a-457b-94ea-318206fa734d
[0916/134938.128206:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134945.809507:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:63769/devtools/browser/3f7fdf88-7630-4f5a-9127-03add4150e68
[0916/134946.247076:INFO:CONSOLE(1)] "%c", source:  (1)
tooturnttony {'diggCount': 158400, 'shareCount': 388, 'commentCount': 1514, 'playCount': 782100}
tooturnttony {'diggCount': 206300, 'shareCount': 4565, 'commentCount': 2570, 'playCount': 1000000}
tooturnttony {'diggCount': 95200, 'shareCount': 809, 'commentCount': 724, 'playCount': 589500}
tooturnttony {'diggCount': 267300, 'shareCount': 1768, 'commentCount': 2225, 'playCount': 1400000}
tooturnttony {'diggCount': 41500, 'shareCount': 341, 'commentCount': 490, 'playCount': 504700}
tooturnttony {'diggCount': 729000, 'shareCount': 1876, 'commentCount': 2215, 'playCount': 5600000}
tooturnttony {'diggCount': 81400, 'shareCount': 607, 'commentCount': 658, 'playCount': 825400}
tooturnttony {'diggCount': 159200, 'shareCount': 819, 'commentCount': 784, 'playCount': 1100000}
tooturnttony {'diggCount': 467500, 'shareCount': 3655, 'commentCount': 1563, 'playCount': 2600000}
tooturnttony {'diggCount': 893100, 'shareCount': 14400, 'commentCount': 2993, 'playCount': 4200000}
[0916/134947.032721:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:64022/devtools/browser/a38dcf19-93e2-4643-a99a-6ef9bf529268
[0916/134947.488030:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134948.337400:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:64026/devtools/browser/b5155840-1a48-4fa0-a26e-bb24d5813a41
[0916/134948.767992:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134949.703368:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:64279/devtools/browser/696f15d8-2275-44f1-bbbd-bf2e6d09c63e
[0916/134950.142703:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134950.939734:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:64530/devtools/browser/1f4315b6-367a-48a0-acb8-5b125827339a
[0916/134951.392556:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134952.163229:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:64534/devtools/browser/c3f67b87-cd5a-4438-a2c4-cdbe9017a250
[0916/134952.591638:INFO:CONSOLE(1)] "%c", source:  (1)
annaxsitar {'diggCount': 8060, 'shareCount': 12, 'commentCount': 223, 'playCount': 22500}
annaxsitar {'diggCount': 24100, 'shareCount': 65, 'commentCount': 178, 'playCount': 115500}
annaxsitar {'diggCount': 18700, 'shareCount': 574, 'commentCount': 478, 'playCount': 78800}
annaxsitar {'diggCount': 30200, 'shareCount': 53, 'commentCount': 244, 'playCount': 132700}
annaxsitar {'diggCount': 136000, 'shareCount': 2895, 'commentCount': 782, 'playCount': 553300}
annaxsitar {'diggCount': 52000, 'shareCount': 44, 'commentCount': 266, 'playCount': 210100}
annaxsitar {'diggCount': 109500, 'shareCount': 228, 'commentCount': 339, 'playCount': 386000}
annaxsitar {'diggCount': 175100, 'shareCount': 676, 'commentCount': 578, 'playCount': 726600}
annaxsitar {'diggCount': 94200, 'shareCount': 429, 'commentCount': 367, 'playCount': 342000}
annaxsitar {'diggCount': 49200, 'shareCount': 153, 'commentCount': 275, 'playCount': 244500}
[0916/134953.495606:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:64787/devtools/browser/b947216f-817e-4c90-8505-02a38616dd3a
[0916/134953.932884:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134954.803371:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:65040/devtools/browser/3687d401-54de-46df-bfeb-57b42773b043
[0916/134955.255228:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134956.153778:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:65044/devtools/browser/09fe5d5e-0861-47e4-98ef-65db60c89456
[0916/134956.591189:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/134957.490562:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:65297/devtools/browser/6b7ae0a6-6580-4e60-845d-67f340c4777a
[0916/134957.919140:INFO:CONSOLE(1)] "%c", source:  (1)
tuckerbudzyn {'diggCount': 3400000, 'shareCount': 442700, 'commentCount': 49800, 'playCount': 14900000}
tuckerbudzyn {'diggCount': 217100, 'shareCount': 2062, 'commentCount': 1010, 'playCount': 833300}
tuckerbudzyn {'diggCount': 686300, 'shareCount': 17400, 'commentCount': 4989, 'playCount': 2700000}
tuckerbudzyn {'diggCount': 305000, 'shareCount': 5470, 'commentCount': 2096, 'playCount': 1800000}
tuckerbudzyn {'diggCount': 85900, 'shareCount': 1495, 'commentCount': 1101, 'playCount': 559700}
tuckerbudzyn {'diggCount': 102200, 'shareCount': 783, 'commentCount': 1248, 'playCount': 607000}
tuckerbudzyn {'diggCount': 370600, 'shareCount': 1741, 'commentCount': 2030, 'playCount': 1600000}
tuckerbudzyn {'diggCount': 527700, 'shareCount': 6486, 'commentCount': 1753, 'playCount': 2000000}
tuckerbudzyn {'diggCount': 116800, 'shareCount': 545, 'commentCount': 713, 'playCount': 688500}
tuckerbudzyn {'diggCount': 257100, 'shareCount': 4750, 'commentCount': 1605, 'playCount': 1100000}
[0916/134958.752228:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:49168/devtools/browser/91c39d50-1dd4-4fca-8b47-5a7f5f8a66b4
[0916/134959.199514:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/135000.047061:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:49173/devtools/browser/22691c9d-7c85-41e7-95af-0760a4392896
[0916/135000.477682:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/135001.273515:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:49425/devtools/browser/d806c7b0-014b-414a-86d5-89fd82f0d30e
[0916/135001.711897:INFO:CONSOLE(1)] "%c", source:  (1)
daquan {'diggCount': 45300, 'shareCount': 3083, 'commentCount': 728, 'playCount': 177500}
daquan {'diggCount': 39500, 'shareCount': 1953, 'commentCount': 621, 'playCount': 170600}
daquan {'diggCount': 50100, 'shareCount': 425, 'commentCount': 307, 'playCount': 231100}
daquan {'diggCount': 59100, 'shareCount': 2798, 'commentCount': 419, 'playCount': 357600}
daquan {'diggCount': 289200, 'shareCount': 10200, 'commentCount': 1469, 'playCount': 1100000}
daquan {'diggCount': 173700, 'shareCount': 5492, 'commentCount': 1251, 'playCount': 810400}
daquan {'diggCount': 72700, 'shareCount': 2280, 'commentCount': 438, 'playCount': 285400}
daquan {'diggCount': 487200, 'shareCount': 7444, 'commentCount': 1533, 'playCount': 1900000}
daquan {'diggCount': 195900, 'shareCount': 2614, 'commentCount': 1438, 'playCount': 795300}
daquan {'diggCount': 397300, 'shareCount': 3436, 'commentCount': 1634, 'playCount': 1800000}
[0916/135002.514178:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:49679/devtools/browser/a89c6d78-0441-496b-8ac9-f9e3b4adda69
[0916/135002.958106:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/135003.728944:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:49683/devtools/browser/ff02bdd5-1a53-4f3f-af1a-a4b6c38f391e
[0916/135004.163449:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/135005.096721:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:49936/devtools/browser/a2de16b8-6006-4989-9028-6bf4beef35e3
[0916/135005.548739:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/135006.340548:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:49940/devtools/browser/c162da31-bdc1-497d-89ab-27f13c6dade7
[0916/135006.774004:INFO:CONSOLE(1)] "%c", source:  (1)
donaldducc {'diggCount': 34800, 'shareCount': 155, 'commentCount': 338, 'playCount': 243000}
donaldducc {'diggCount': 29200, 'shareCount': 169, 'commentCount': 302, 'playCount': 218100}
donaldducc {'diggCount': 15700, 'shareCount': 98, 'commentCount': 201, 'playCount': 86300}
donaldducc {'diggCount': 38500, 'shareCount': 133, 'commentCount': 384, 'playCount': 206900}
donaldducc {'diggCount': 39600, 'shareCount': 134, 'commentCount': 226, 'playCount': 242100}
donaldducc {'diggCount': 36700, 'shareCount': 258, 'commentCount': 355, 'playCount': 193500}
donaldducc {'diggCount': 43400, 'shareCount': 183, 'commentCount': 336, 'playCount': 409100}
donaldducc {'diggCount': 37800, 'shareCount': 182, 'commentCount': 246, 'playCount': 255700}
donaldducc {'diggCount': 16600, 'shareCount': 80, 'commentCount': 173, 'playCount': 109700}
donaldducc {'diggCount': 115700, 'shareCount': 519, 'commentCount': 482, 'playCount': 772600}
[0916/135007.551540:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:50193/devtools/browser/3f8f9c38-0cbf-4e59-b2c9-f49472a663aa
[0916/135008.007908:INFO:CONSOLE(1)] "%c", source:  (1)
[0916/135008.953695:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.

DevTools listening on ws://127.0.0.1:50197/devtools/browser/ee4f7086-b646-468b-9e6a-f3588ad8b55b
[0916/135009.385314:INFO:CONSOLE(1)] "%c", source:  (1)
Traceback (most recent call last):
  File "snowball.py", line 21, in <module>
    tiktok = api.getUser(accounts)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/TikTokApi/tiktok.py", line 723, in getUser
    return self.getData(b, proxy=proxy)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/TikTokApi/tiktok.py", line 71, in getData
    query = {'verifyFp': b.verifyFp, 'did': b.did, '_signature': b.signature}
AttributeError: 'browser' object has no attribute 'verifyFp'
```

**Comment by edenhikri** [comment]
I discovered the issue may have been with google-api-core interfering with requests.

I simple fixed it by:
```
pip3 uninstall requests
```
then:
```
pip3 uninstall google-api-core
```
finally:
```
pip3 install requests
````

Thank you kindly for your help! @davidteather @fastandslow007

**Comment by edenhikri** [comment]
Unfortunately, this did not actually fix the issue. It seemed to only have fixed the issue for a few hours... I am really not sure what is causing this issue. Can you please help me out @davidteather

**Comment by davidteather** [comment]
I’ve got no clue what your issue is. You could try to make a VM for a fresh start. The .travis.yml file has specific setup commands

**Comment by edenhikri** [comment]
Those instructions are linux specific... are you able to provide Mac instructions?

**Comment by edenhikri** [comment]
I am really unsure what the issue may be... I have tried installing a new virtual environment, running it with debug mode set to True and dumpio enabled on launch. Nothing seems to work. I would love to use this program however it does not work for me

**Comment by davidteather** [comment]
@edenhikri might be an issue with your actual IP address might be blocked by TikTok. When I used heroku it didn't like how they had their IPs setup. When I had a dedicated IP on a AWS ec2 instance it worked. You might want to try making the requests over a proxy (most publically available free ones probably wont work)

**Comment by edenhikri** [comment]
What type of proxy do you recommend or do you have any good websites for purchasing proxies that will work?

**Comment by edenhikri** [comment]
@davidteather How do I authenticate the proxy I am using with username and password?

**Comment by edenhikri** [comment]
@davidteather So I tried using a private proxy that I bought based in United States and ran the script in debug mode in a virtual environment and I still got the same error:
```
  File "pinpoint.py", line 164, in pullandsort
    description()
  File "pinpoint.py", line 80, in description
    tiktok = api.getUser(name, proxy=myproxy)
  File "/Users/edenhikri/Desktop/ANYMERCH/programming/pinpoint/venv/lib/python3.8/site-packages/TikTokApi/tiktok.py", line 723, in getUser
    return self.getData(b, proxy=proxy)
  File "/Users/edenhikri/Desktop/ANYMERCH/programming/pinpoint/venv/lib/python3.8/site-packages/TikTokApi/tiktok.py", line 71, in getData
    query = {'verifyFp': b.verifyFp, 'did': b.did, '_signature': b.signature}
AttributeError: 'browser' object has no attribute 'verifyFp'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "pinpoint.py", line 231, in <module>
    startup()
  File "pinpoint.py", line 227, in startup
    locate()
  File "pinpoint.py", line 49, in locate
    sort()
  File "pinpoint.py", line 71, in sort
    pullandsort()
  File "pinpoint.py", line 167, in pullandsort
    description()
  File "pinpoint.py", line 80, in description
    tiktok = api.getUser(name, proxy=myproxy)
  File "/Users/edenhikri/Desktop/ANYMERCH/programming/pinpoint/venv/lib/python3.8/site-packages/TikTokApi/tiktok.py", line 723, in getUser
    return self.getData(b, proxy=proxy)
  File "/Users/edenhikri/Desktop/ANYMERCH/programming/pinpoint/venv/lib/python3.8/site-packages/TikTokApi/tiktok.py", line 71, in getData
    query = {'verifyFp': b.verifyFp, 'did': b.did, '_signature': b.signature}
AttributeError: 'browser' object has no attribute 'verifyFp'
```

**Comment by edenhikri** [comment]
When I run the program with `debug=True`
 and with `self.dumpio = options.get('dumpio', True)` 
which is in `.../pyppeteer/launcher.py` I get this error:

```
Traceback (most recent call last):
  File "/Users/edenhikri/Desktop/ANYMERCH/programming/pinpoint/venv/lib/python3.8/site-packages/TikTokApi/browser.py", line 72, in __init__
    self.loop = asyncio.new_event_loop()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/events.py", line 758, in new_event_loop
    return get_event_loop_policy().new_event_loop()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/events.py", line 656, in new_event_loop
    return self._loop_factory()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/unix_events.py", line 54, in __init__
    super().__init__(selector)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/selector_events.py", line 61, in __init__
    self._make_self_pipe()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/selector_events.py", line 108, in _make_self_pipe
    self._ssock, self._csock = socket.socketpair()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/socket.py", line 571, in socketpair
    a, b = _socket.socketpair(family, type, proto)
OSError: [Errno 24] Too many open files

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "pinpoint.py", line 164, in pullandsort
    description()
  File "pinpoint.py", line 80, in description
    tiktok = api.getUser(name, proxy=myproxy)
  File "/Users/edenhikri/Desktop/ANYMERCH/programming/pinpoint/venv/lib/python3.8/site-packages/TikTokApi/tiktok.py", line 722, in getUser
    b = browser(api_url, proxy=proxy)
  File "/Users/edenhikri/Desktop/ANYMERCH/programming/pinpoint/venv/lib/python3.8/site-packages/TikTokApi/browser.py", line 80, in __init__
    self.loop.close()
AttributeError: 'browser' object has no attribute 'loop'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/edenhikri/Desktop/ANYMERCH/programming/pinpoint/venv/lib/python3.8/site-packages/TikTokApi/browser.py", line 72, in __init__
    self.loop = asyncio.new_event_loop()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/events.py", line 758, in new_event_loop
    return get_event_loop_policy().new_event_loop()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/events.py", line 656, in new_event_loop
    return self._loop_factory()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/unix_events.py", line 54, in __init__
    super().__init__(selector)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/selector_events.py", line 61, in __init__
    self._make_self_pipe()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/selector_events.py", line 108, in _make_self_pipe
    self._ssock, self._csock = socket.socketpair()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/socket.py", line 571, in socketpair
    a, b = _socket.socketpair(family, type, proto)
OSError: [Errno 24] Too many open files

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "pinpoint.py", line 231, in <module>
    startup()
  File "pinpoint.py", line 227, in startup
    locate()
  File "pinpoint.py", line 49, in locate
    sort()
  File "pinpoint.py", line 71, in sort
    pullandsort()
  File "pinpoint.py", line 167, in pullandsort
    description()
  File "pinpoint.py", line 80, in description
    tiktok = api.getUser(name, proxy=myproxy)
  File "/Users/edenhikri/Desktop/ANYMERCH/programming/pinpoint/venv/lib/python3.8/site-packages/TikTokApi/tiktok.py", line 722, in getUser
    b = browser(api_url, proxy=proxy)
  File "/Users/edenhikri/Desktop/ANYMERCH/programming/pinpoint/venv/lib/python3.8/site-packages/TikTokApi/browser.py", line 80, in __init__
    self.loop.close()
AttributeError: 'browser' object has no attribute 'loop'
Exception ignored in: <function BaseEventLoop.__del__ at 0x10f3ac5e0>
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/base_events.py", line 656, in __del__
    self.close()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/unix_events.py", line 58, in close
    super().close()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/selector_events.py", line 92, in close
    self._close_self_pipe()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/selector_events.py", line 99, in _close_self_pipe
    self._remove_reader(self._ssock.fileno())
AttributeError: '_UnixSelectorEventLoop' object has no attribute '_ssock'
Exception ignored in: <function BaseEventLoop.__del__ at 0x10f3ac5e0>
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/base_events.py", line 656, in __del__
    self.close()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/unix_events.py", line 58, in close
    super().close()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/selector_events.py", line 92, in close
    self._close_self_pipe()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/selector_events.py", line 99, in _close_self_pipe
    self._remove_reader(self._ssock.fileno())
AttributeError: '_UnixSelectorEventLoop' object has no attribute '_ssock'
```

**Comment by edenhikri** [comment]
Is there anything else I can try? @davidteather

**Comment by fastandslow007** [comment]
Hi,

Sorry to jump in again.

“ OSError: [Errno 24] Too many open files”

Have you try to increase open files via sysctl.conf or limits.conf or
ulimit?

**Comment by edenhikri** [comment]
@fastandslow007 I am on a Mac... where are these files located?

**Comment by fastandslow007** [comment]
might depend on your osx version as well.
but lauchd and ulimit should do.

I found by simply googling: "macos too many open files".
https://superuser.com/questions/830149/os-x-yosemite-too-many-files-open
https://medium.com/mindful-technology/too-many-open-files-limit-ulimit-on-mac-os-x-add0f1bfddde

**Comment by AlexB51** [comment]
If this is still an issue for you, I had the same problem. I added the following and it resolved the issue with set_async:

from TikTokApi.browser import set_async
set_async()

Good luck, and let me know!

**Comment by lo-fo** [comment]
same issue here.

**Comment by Danielohayon** [comment]
> If this is still an issue for you, I had the same problem. I added the following and it resolved the issue with set_async:
> 
> from TikTokApi.browser import set_async
> set_async()
> 
> Good luck, and let me know!

I had the same problem and this solved it.
Thanks!

**Comment by vasemkin** [comment]
i encountered the same issue.
simply running `pyppeteer-install` was the solution for me.

**Comment by hanqingguo** [comment]
> > If this is still an issue for you, I had the same problem. I added the following and it resolved the issue with set_async:
> > from TikTokApi.browser import set_async
> > set_async()
> > Good luck, and let me know!
> 
> I had the same problem and this solved it.
> Thanks!

Same issue and solved by this!

**Comment by Incher25** [comment]
> If this is still an issue for you, I had the same problem. I added the following and it resolved the issue with set_async:
> 
> from TikTokApi.browser import set_async
> set_async()
> 
> Good luck, and let me know!

I have the same issue with "verifyFp" but after adding set_async() I just have anorher error:

> Error in atexit._run_exitfuncs:
> Traceback (most recent call last):
>   File "/usr/lib/python3.6/asyncio/base_events.py", line 454, in run_until_complete
>     self.run_forever()
>   File "/usr/lib/python3.6/asyncio/base_events.py", line 408, in run_forever
>     raise RuntimeError('This event loop is already running')
> RuntimeError: This event loop is already running

**Comment by WinkyDev** [comment]
> > If this is still an issue for you, I had the same problem. I added the following and it resolved the issue with set_async:
> > from TikTokApi.browser import set_async
> > set_async()
> > Good luck, and let me know!
> 
> I have the same issue with "verifyFp" but after adding set_async() I just have anorher error:
> 
> > Error in atexit._run_exitfuncs:
> > Traceback (most recent call last):
> > File "/usr/lib/python3.6/asyncio/base_events.py", line 454, in run_until_complete
> > self.run_forever()
> > File "/usr/lib/python3.6/asyncio/base_events.py", line 408, in run_forever
> > raise RuntimeError('This event loop is already running')
> > RuntimeError: This event loop is already running

Same error here

**Comment by gopida** [comment]
I had the same issue after deploying it to my raspberry pi, whereas it is working perfectly in windows. 
Looks like the issue is with the pyppeteer browser. Did not investigate it completely, but the **below fixed the issue.** 

**Fix**

pass the below argument while creating TikTokApi() object
_executablePath = '/usr/bin/chromium-browser'_

Command to find the path - "whereis chromium-browser"

**Alternative:**
Or directly hardcode the browser path in site-packages/TikTokApi/browser.py (wouldn't recommend, but can give a try if the 1st doesn't work)
_self.executablePath = '/usr/bin/chromium-browser'_

Hopefully, this works in Linux as well.

**Comment by DeadboyTimmy** [comment]
I've tried all of your recommendations, but the problem remained unresolved
Here's the code with all libraries:
`from TikTokApi import TikTokApi
from TikTokApi.browser import set_async
set_async()
api = TikTokApi()

results = 10

trending = api.trending(count=results)

for tiktok in trending:
    # Prints the text of the tiktok
    print(tiktok['desc'])

print(len(trending))`
And here's some errors:
`[W:pyppeteer.chromium_downloader] start chromium download.
Download may take a few minutes.
Traceback (most recent call last):
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1108)

During handling of the above exception, another exception occurred:
`
How can I solve it on MacOS?

**Comment by minakraka** [comment]
have You tried this?
[https://stackoverflow.com/questions/57250102/python3-ssl-certificate-problem-when-requests-html-install-chromium-using-pyppet]

**Comment by gopida** [comment]
> ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED]
The issue you are facing seem to be different than the one discussed in this thread. [See this ](https://stackoverflow.com/questions/42098126/mac-osx-python-ssl-sslerror-ssl-certificate-verify-failed-certificate-verify) for possible solution for ssl issue.

**Comment by davidteather** [comment]
Since I've migrated the API to playwright information here may be outdated for TikTokApi >= 3.7.0 if anyone has the same issues on 3.7.0 or greater please open a new issue.

**Comment by viniciusao** [comment]
> What happens when you type the following command into your terminal
> 
> ```
> chromedriver
> ```

I was getting an error that TikTokApi didn't have an attribute (width). When I saw your post, it enlighted me. I didn't know I should had started chromedriver x)! Thanks, mate.
